plugins/vpc-shared-eni/network/bridge_windows.go

```diff
@@ -36,6 +37,18 @@ const (
 	hnsEndpointNameFormat = "cid-%s"
 )
 
+// NSType identifies the namespace type for the containers.
+type NSType int
+
+const (
+	// infraContainerNS identifies an Infra container NS for networking setup.
+	infraContainerNS NSType = iota
+	// nonInfraContainerNS identifies sharing of infra container NS for networking setup.
+	nonInfraContainerNS
```

Comment:
In hindsight, I think we should have called this enum value by what it represents, instead of what it is not. So perhaps "appContainerNS" is a better name?